### PR TITLE
Laser pointer fixes

### PR DESCRIPTION
**Pull request by Casper3667**
This adds a delay to using the laser pointer  and gives everyone a chat message that it was used and on what. Additionally the dot it creates lasts a little longer.

### CODE REVIEW

**Comment by Casper3667** [comment]
!review

**Comment by Doxxmedearly** [comment]
Would appreciate throwing \the checks in front of mentions of users and targets, even if they were not present before.

**Review by Geevies — APPROVED** [review]

File: code/game/objects/items/devices/laserpointer.dm [hunk 1]
```diff
@@ -64,12 +65,13 @@
...
 		return
 	laser_act(target, user)
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
 
 /obj/item/device/laser_pointer/proc/laser_act(var/atom/target, var/mob/living/user)
 	if( !(user in (viewers(7,target))) )
 		return
 	if (!diode)
-		to_chat(user, "<span class='notice'>You point [src] at [target], but nothing happens!</span>")
+		to_chat(user, "<span class='notice'>You point [src] at \the [target], but nothing happens!</span>")
```
Comment (Doxxmedearly) [inline code/game/objects/items/devices/laserpointer.dm:74]:
```suggestion
		to_chat(user, "<span class='notice'>You point \the [src] at \the [target], but nothing happens!</span>")
```
Just missed this one.

Comment (Casper3667) [reply code/game/objects/items/devices/laserpointer.dm]: There we go, thanks for spotting that.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**